src/AssemblerHttpServer-Alignments.cpp

```diff
@@ -1182,6 +1183,262 @@ void Assembler::computeAllAlignments(
+    const bool maxLengthIsPresent = getParameterValue(request, "maxLength", maxLength);
+    showAlignmentResults = getParameterValue(request, "showAlignmentResults", showAlignmentResultsString);
+
+    // Get alignment parameters.
+    computeAllAlignmentsData.method = httpServerData.assemblerOptions->alignOptions.alignMethod;
+    getParameterValue(request, "method", computeAllAlignmentsData.method);
+    computeAllAlignmentsData.minMarkerCount = 0;
+    getParameterValue(request, "minMarkerCount", computeAllAlignmentsData.minMarkerCount);
+    computeAllAlignmentsData.maxSkip = httpServerData.assemblerOptions->alignOptions.maxSkip;
+    getParameterValue(request, "maxSkip", computeAllAlignmentsData.maxSkip);
+    computeAllAlignmentsData.maxDrift = httpServerData.assemblerOptions->alignOptions.maxDrift;
+    getParameterValue(request, "maxDrift", computeAllAlignmentsData.maxDrift);
+    computeAllAlignmentsData.maxMarkerFrequency = httpServerData.assemblerOptions->alignOptions.maxMarkerFrequency;
+    getParameterValue(request, "maxMarkerFrequency", computeAllAlignmentsData.maxMarkerFrequency);
+    computeAllAlignmentsData.minAlignedMarkerCount = httpServerData.assemblerOptions->alignOptions.minAlignedMarkerCount;
+    getParameterValue(request, "minAlignedMarkerCount", computeAllAlignmentsData.minAlignedMarkerCount);
+    computeAllAlignmentsData.minAlignedFraction = httpServerData.assemblerOptions->alignOptions.minAlignedFraction;
+    getParameterValue(request, "minAlignedFraction", computeAllAlignmentsData.minAlignedFraction);
+    computeAllAlignmentsData.maxTrim = httpServerData.assemblerOptions->alignOptions.maxTrim;
+    getParameterValue(request, "maxTrim", computeAllAlignmentsData.maxTrim);
+    computeAllAlignmentsData.matchScore = httpServerData.assemblerOptions->alignOptions.matchScore;
+    getParameterValue(request, "matchScore", computeAllAlignmentsData.matchScore);
+    computeAllAlignmentsData.mismatchScore = httpServerData.assemblerOptions->alignOptions.mismatchScore;
+    getParameterValue(request, "mismatchScore", computeAllAlignmentsData.mismatchScore);
+    computeAllAlignmentsData.gapScore = httpServerData.assemblerOptions->alignOptions.gapScore;
+    getParameterValue(request, "gapScore", computeAllAlignmentsData.gapScore);
+    computeAllAlignmentsData.downsamplingFactor = httpServerData.assemblerOptions->alignOptions.downsamplingFactor;
+    getParameterValue(request, "downsamplingFactor", computeAllAlignmentsData.downsamplingFactor);
+    computeAllAlignmentsData.bandExtend = httpServerData.assemblerOptions->alignOptions.bandExtend;
+    getParameterValue(request, "bandExtend", computeAllAlignmentsData.bandExtend);
+
+    html << "<h1>Alignment statistics</h1>";
+    html << "<p>This page enables sampling from the pool of reads and computing alignments for each read in the"
+            "sample. Once alignment finishes, stats can be generated and used to evaluate Shasta parameters."
```

Comment:
For clarity I would change the message to "...for each read in the sample against all other reads in this assembly. This can be slow."